build-packages/check-pr/validators.ts

```diff
@@ -127,7 +127,7 @@ export function validateChangesets(
   }
 
   const changeTypes = fileContents.flatMap(content => {
-    const matches = content.match(/\[([^\]]+)\]/g);
+    const matches = content.match(/\[([^\]]+)\](?!\([^\]]*\))/g);
```

Comment:
[For reviewer] The `check-pr` workflow is failing because it is matching `text` from `[text](http://example.com)` as preamble.. I fixed it here but since it is not merged, the workflow still fails..